turbinia/task_manager.py

```diff
@@ -95,8 +97,15 @@ def task_runner(obj, *args, **kwargs):
   Returns:
     Output from TurbiniaTask (should be TurbiniaTaskResult).
   """
-  obj = workers.TurbiniaTask.deserialize(obj)
-  return obj.run_wrapper(*args, **kwargs)
+  if os.path.exists(config.LOCK_FILE) or os.path.exists(config.TO_BE_DELETED):
+    raise psq.Retry()
```

Comment:
Can you add a log line here?

Reply:
From testing its super super spammy and will flood the logs. We can adjust this by adjusting PSQ_QUEUE_WAIT_SECONDS, but still spammy + longer we wait -> longer everything will take to run.

Reply:
Oh, interesting, I assumed that it would only happen once, but maybe the same worker keeps picking up the same task for a while until another one actually picks it up?

Reply:
yeah I think that could be so, hard to see in debugger but what I did as a test was kill the given worker after its completed a Task and then restart a new worker to see it got Tasks down to 0 also spinning up another worker locally now seems to pick up work while another worker is executing something, which didn't seem to do that before